rootfs/templates/confd_settings.py

```diff
@@ -16,6 +16,8 @@
 
 {{ if exists "/deis/controller/registrationMode" }}
 REGISTRATION_MODE = '{{ getv "/deis/controller/registrationMode" }}'
+{{ else }}
+REGISTRATION_MODE = 'enabled'
```

Comment:
+1. Either that or make this configurable in deis/settings.py via `REGISTRATION_MODE = os.getenv('REGISTRATION_MODE', 'enabled')`